facts_outputs.Rmd

```diff
@@ -0,0 +1,59 @@
+---
+title: "Comparing FACTS Outputs"
+author: "Ciara Donegan"
+date: "2024-09-19"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+knitr::opts_knit$set(root.dir = file.path("data","outputs","fair_results"))
+library(tidyverse)
+library(ggplot2)
+library(ncdf4)
+```
+
+## FACTS Outputs
+
+This document compares the outputs of various FACTS experiments, with the original climate input from the FaIR simple climate model and with added input from Hector *(to be added)*.
+
```

Comment:
For every experiment & SCM, you will need to repeat the same lines of code for opening the nc file, extracting the data, and getting the summary results. Do you think you could write a function to help do this to minimize the amount of code you have to write for the following experiments?